### ISSUE TITLE
Can't tirgger radio onChange event in Filed Component

### ISSUE THREAD

**Issue by wd2010** [erikras/redux-form#2448]
My Filed Component is:
```jsx
<Field  
    name="rechargeType" 
    component="input"
    type="radio"
    value={item.text}
   onChange={::this.handleChange}				
/>
```

and the handleChange Fun is:
```jsx
    handleChange(e){
        debugger
        console.log(e.target)
    }
```

**the handleChange has no work,and I try to set the Field Component into:** 
```js
<input  
    name="rechargeType" 
    type="radio"
    value={item.text}
   onChange={::this.handleChange}				
/>
```
**and the fun of handleChange can work, but I must use <Filed ... /> to connect redux and via change the value of radio to change other thing, how can I do?**

**Comment by sebastianoblom** [comment]
I had similar problems about the onChange event. Dont know if it is intended or not.
You can manually override the Field components onChange, like this #2406 - I am new to this lib though, so dont know if this is the proper way to do it.

I also found this #2434 today so seems other people having problems with this too.

Perhaps there should be some documentation/examples about how to deal with custom onChange events? Afterall its quite common to do more stuff than just set the value for an onChange.

**Comment by gustavohenke** [comment]
Fixed by #2385. Let's wait for a new release.
